bb-practice.html

```diff
@@ -0,0 +1,220 @@
+                </div>
+                <div class="column">
+                    <img src=" /images/red-circle-white-flower.svg" alt="red circle with white flower">
+                    <h4>BlackBerry Spark</h4>
+                    <img src=" /images/wave-dots.jpg" alt="wave of blue dots">
+                    <h3>Zero Trust Meets Zero Touch</h3>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
+                    <button>LEARN MORE</button>
+                </div>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <h4>Spotlight on Innovation</h4>
+                </div>
+                <div class="column">                    
+                    <img src=" /images/2021-report-img.jpg" alt="image of threat report">                   
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                    <button>DOWNLOAD REPORT</button>
+                </div>
+                <div class="column">
+                    <img src=" /images/hand-closeup-mobile-screen.jpg" alt="close up image of hand holding a cell phone">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                    <button>DOWNLOAD REPORT</button>
+                </div>
+                <div class="column">
+                    <img src=" /images/black-woman-on-laptop.jpg" alt="black woman on laptop">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                    <button>DOWNLOAD REPORT</button>
+                </div>
+                <div class="column">
+                    <img src=" /images/graphic-hamut.jpg" alt="red background image of a BA hamut poster">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                    <button>DOWNLOAD REPORT</button>
+                </div>
+                <div class="column">
```

Comment:
I don't believe this social media column belongs within this row.
This is an excellent case where in the `<footer>`, you have multiple rows.
See my bible for example code.

Reply:
On the BlackBerry page there is a horizontal line separating this section from footer content.

Reply:
The footer can contain more than navigational links to points on a website.
Two ways to look at a footer:
1) Generally, a footer on a page will contain links, social media links, copyright, etc.
2) Whatever would appear over and over from page to page at the bottom.